### ISSUE TITLE
Log a message if `npm install` is taking too long to finish

### ISSUE THREAD

**Issue by tarekoraby** [vaadin/flow#13070]
### Describe your motivation

Especially on slow machines and without `package-lock.json`, it might seem to the user running a Vaadin project that the build has frozen sometime after `npm install` has started. 

### Describe the solution you'd like

If `npm install` is taking too long to finish, log some message that informs the user that something is still ongoing. And if appropriate, the message should mention that the delay is a one-time issue due to a missing `package-lock.json`.

**Comment by Artur-** [comment]
Was this fixed by #13047?

**Comment by knoobie** [comment]
What do you guys think about showing the npm output? So the user sees that something is happening?

**Comment by Artur-** [comment]
Is it not shown?  I see
```
2022-02-17 15:17:51.596  INFO 93022 --- [onPool-worker-1] c.v.f.s.frontend.TaskUpdatePackages      : using '/.../node/v17.3.0/bin/npm --no-update-notifier --no-audit --scripts-prepend-node-path=true --ignore-scripts install' for frontend package installation
⸨⠂⠂⠂⠂⠂⠂⠂⠂⠂⠂⠂⠂⠂⠂⠂⠂⠂⠂⸩ ⠋ idealTree:start: sill idealTree buildDeps
```

**Comment by knoobie** [comment]
You are right's - it's there! For me it's fast with package-lock that's why I probably never really noticed.. I was hoping it was missing and enabling it would be a quick fix to show long running tasks

**Comment by tarekoraby** [comment]
> Was this fixed by #13047?

Yeah, it seems so. I would have rather logged the message when the installation actually started. Somewhere after this [line](https://github.com/vaadin/flow/blob/45082c1cf0eacffdf10033b21e9e6270728b3a89/flow-server/src/main/java/com/vaadin/flow/server/frontend/TaskRunNpmInstall.java#L327). But that can be done in a separate PR.

**Comment by javier-godoy** [comment]
> Is it not shown? I see
> 
> ```
> 2022-02-17 15:17:51.596  INFO 93022 --- [onPool-worker-1] c.v.f.s.frontend.TaskUpdatePackages      : using '/.../node/v17.3.0/bin/npm --no-update-notifier --no-audit --scripts-prepend-node-path=true --ignore-scripts install' for frontend package installation
> ⸨⠂⠂⠂⠂⠂⠂⠂⠂⠂⠂⠂⠂⠂⠂⠂⠂⠂⠂⸩ ⠋ idealTree:start: sill idealTree buildDeps
> ```

So it seems it's an issue with the IDE.  Compiling from console shows npm output as @Artur- says (although it's quiet if pnpm is used). With Eclipse (2021-09) there is no ouput even when using npm, thus logging a message as @tarekoraby suggested is a good idea.

```
2022-02-17 15:33:46.344  INFO 22412 --- [nio-8080-exec-1] o.s.web.servlet.DispatcherServlet        : Completed initialization in 0 ms
2022-02-17 15:33:46.412  INFO 22412 --- [nio-8080-exec-1] c.vaadin.flow.spring.SpringInstantiator  : The number of beans implementing 'I18NProvider' is 0. Cannot use Spring beans for I18N, falling back to the default behavior
npm WARN read-shrinkwrap This version of npm is compatible with lockfileVersion@1, but package-lock.json was generated for lockfileVersion@2. I'll try to do my best with it!
npm WARN no-name@ No repository field.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.13 (node_modules\watchpack-chokidar2\node_modules\fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.13: wanted {"os":"darwin","arch":"any"} (current: {"os":"win32","arch":"x64"})
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.3.2 (node_modules\fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.3.2: wanted {"os":"darwin","arch":"any"} (current: {"os":"win32","arch":"x64"})
2022-02-17 15:36:42.144  INFO 22412 --- [onPool-worker-3] c.v.f.s.frontend.TaskUpdatePackages      : Frontend dependencies resolved successfully.
```
